src/components/Sidebar/styles.scss

```diff
@@ -5,7 +5,7 @@
   position: fixed;
   height: 100%;
   background-color: $secondary-color;
-  width: 302px;
```

Comment:
change it to 
`max-width: 302px;`

Reply:
actually, can you not modify `SideBar` styles? I already made some changes on this on my change